src/openvic-simulation/pop/Pop.hpp

```diff
@@ -73,6 +79,8 @@ namespace OpenVic {
 	struct PopType : HasIdentifierAndColour {
 		friend struct PopManager;
 
+		enum struct income_type_t : uint8_t { NONE, ADMINISTRATION, EDUCATION, MILITARY, REFORMS };
```

Comment:
bit flags?

Reply:
If they can't be used simulatenously sure, for now I'm storing them in a vector to allow for repeats

Reply:
Each type can only be used once. Bit flags seems a great solution for that.